src/WebJobs.Script.WebHost/Controllers/Results/FileCallbackResult.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Internal;
+using Microsoft.Extensions.DependencyInjection;
```

Comment:
All our other result types are organized under the existing Models folder

Reply:
hmmm, I recall @fabiocav had a custom Result before in this folder but looking at the history, I'm probably wrong. I'll move it.